gradle/gradle-mvn-push.gradle

```diff
@@ -85,6 +85,7 @@ afterEvaluate { project ->
     task androidJavadocs(type: Javadoc) {
         source = android.sourceSets.main.java.srcDirs
         classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
+        excludes = ['**/*.kt']
```

Comment:
so this fixed the issue?

Reply:
Yes, basically ignoring Kotlin files for javadoc generation.